libs/watt/package/icon/icons.ts

```diff
@@ -85,4 +86,9 @@ export const WattIconMap = {
   horizontalRule: 'horizontal_rule',
   wrongLocation: 'wrong_location',
   heatPump: 'heat_pump',
+  inventory: 'inventory',
+  globe: 'globe',
+  calculate: 'calculate',
+  bar_chart_4_bars: 'bar_chart_4_bars',
+  view_list: 'view_list',
 };
```

Comment:
suggestion: let's show the icons in Storybook as well. Done by adding them to `storybook-icon-overview.component.ts`